src/backend/gl/glx.c

```diff
@@ -523,6 +523,7 @@ static void glx_diagnostics(backend_t *base) {
 struct backend_operations glx_ops = {
     .init = glx_init,
     .deinit = glx_deinit,
+    .root_change = gl_root_change,
```

Comment:
Can you also add this to the egl backend?

Reply:
I already did that in egl.c